src/query-builder/InsertQueryBuilder.ts

```diff
@@ -371,6 +372,7 @@ export class InsertQueryBuilder<Entity> extends QueryBuilder<Entity> {
             | string[],
         conflictTarget?: string | string[],
         orUpdateOptions?: InsertOrUpdateOptions,
+        conflictConditions?: string[],
```

Comment:
do we have conditions other than "conflict"? If no, better to call this parameter `where`, and I think it can be simply `where?: string`, no need to use arrays here.

Reply:
I did not see any other conditions so I made the type change suggested. Instead of using `where` I used the exact term used in the Postgres documentation [1] (under ON CONFLICT Clause) to provide contextual reference for its usage.

[1] https://www.postgresql.org/docs/current/sql-insert.html